### PR TITLE
[build] Changed default login page

### PR DESCRIPTION
**Pull request by lina**
- Changed some url link names 
- Added some tags and user views on the side bar
- Deleted some redundant links in upper right hand drop down menu
- Added logout link for logout button (upper right hand drop down menu)

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -47,31 +47,54 @@
...
-          <a href="https://github.com/rdash/rdash-angular/README.md" target="_blank">
-            About
+          <a href="http://reactoroverflow.com/creators" target="_blank">
+            Creators
           </a>
         </div>
-        <div class="col-xs-4">
-          <a href="#">
-            Support
+        <div class="https://">
```
Comment (codydaig) [inline client/index.html]: Shouldn't this class still be col-xs-4?

File: client/index.html [hunk 2]
```diff
@@ -95,23 +118,13 @@
...
-                    <a href="#">
-                      Menu Item
+                    <a href="http://www.github.com">
+                      My Github
                     </a>
                   </li>
                   <li class="divider"></li>
                   <li class="link">
-                    <a href="#">
+                    <a href="#/logout">
```
Comment (codydaig) [inline client/index.html]: This link should not include the #. It needs to send the user to the /logout directly on the server.

**Comment by codydaig** [comment]
@linalu1 I made some comments on specific lines in the code. Once those small things are addressed, It will look good to me! :-D Good job!

File: client/index.html (hunk 1 above)
Comment (lina) [inline client/index.html]: Yes. my bad

**Comment by Kevbyte** [comment]
Lgtm

**Comment by codydaig** [comment]
LGTM

**Comment by AndrewKishino** [comment]
Lgtm

**Pull request merged**